android/assets/jsons/Civ V - Vanilla/Units.json

```diff
@@ -1047,7 +1046,8 @@
 		"cost": 1000,
 		"requiredTech": "Rocketry",
 		"requiredResource": "Uranium",
-		"uniques": ["Requires Manhattan Project"],
+		"requiredWonder": "Manhattan Project",
```

Comment:
I would leave this as a unique, but "Requires [Manhattan Project"].
Adding new non-unique fields is not a good idea - it means you need to add a lot of player-facing text to show this information, and it clogs up the base classes, if you see what I mean...
I changed a lot of what were originally fields to uniques, and I think there are even more that could be changed (Interception for example always has range and percentage)...

Reply:
ok, will be redone as unique

Reply:
done